include/multi_blas_helper.cuh

```diff
@@ -80,9 +86,14 @@ namespace quda
        the maximum size of YW is and allocate this amount of space.  This
        allows for a much larger NXZ (NYW) when NYW (NXZ) is small.
     */
-    template <int NXZ, typename SpinorX, typename SpinorY, typename SpinorZ, typename SpinorW, typename Functor>
+    template <int NXZ, typename xType, typename yType, typename Functor>
     inline constexpr int max_YW_size()
     {
+      using SpinorX = Spinor<xType, 4>;
+      using SpinorY = Spinor<yType, 4>;
+      using SpinorZ = SpinorX;
+      using SpinorW = Spinor<xType, 4>;
```

Comment:
just style: why not use SpinorW=SpinorX here as well (as for SpinorZ) ?

Reply:
I guess I was tired.  Will push a quick fix for this.